ArkEcosystem/explorer
# feat: home page statistics box

## Summary

Depends on:
- https://github.com/ArkEcosystem/laravel-ui/pull/473
- https://app.clickup.com/t/jd87rd

Add the home page statistic box:

### To test

- You can set your `.env` settings  as `EXPLORER_NETWORK=production` and `EXPLORER_NETWORK=development` 
- Test dark mode and different screen sizes

Another points to focus on the test:
- That the statistics values are correct
- The refresh time/cache is what is expected, for example, the chart is updated every minute and values are cached maybe we want something more real-time so I leave that to your more experienced opinion.


## Checklist
